src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 2157;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const addTodo = (newTodo: Omit<Todo, 'id' | 'userId'>) => {
+  return client.post<Todo>(`/todos`, { ...newTodo, userId: USER_ID });
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const updateTodo = (todo: Todo) => {
+  return client.patch<Todo>(`/todos/${todo.id}`, todo);
+};
```

Comment:
ur path is duplicated, can we put it in a variable?
```suggestion
export const getTodos = () => {
  return client.get<Todo[]>(`${EXAMPLE_API_PATH}?userId=${USER_ID}`);
};

export const addTodo = (newTodo: Omit<Todo, 'id' | 'userId'>) => {
  return client.post<Todo>(`${EXAMPLE_API_PATH}`, { ...newTodo, userId: USER_ID });
};

export const deleteTodo = (todoId: number) => {
  return client.delete(`${EXAMPLE_API_PATH}/${todoId}`);
};

export const updateTodo = (todo: Todo) => {
  return client.patch<Todo>(`${EXAMPLE_API_PATH}/${todo.id}`, todo);
};
```